src/game_inventory.cpp

```diff
@@ -487,6 +487,26 @@ item_location game_menus::inv::disassemble( Character &p )
                          _( "You don't have any items you could disassemble." ) );
 }
 
+
+/*CANNOT BE PLACED AT CONSUMPTION.CPP AS IT REQUIRES CONVERT_VOLUME, NOT DEFINED HERE.
+PLEASE PLACE THIS SOMEWHERE MORE APPROPRIATE, WHERE CONVERT_VOLUME IS DEFINED*/
```

Comment:
The `convert_volume` function is defined in `src/units_utility.h`, so you should be able to put this function in `src/consumption.cpp` as long as you have an `#include "units_utility.h"` at the top of that file with the other includes.

Keep in mind `convert_volume` returns a value based on the user's configured `VOLUME_UNITS` (in cups, liters, or quarts), which will affect the return value, in turn affecting how the bar-graph looks - not sure if that's what you intended. But now that you have factored it out into a separate function, this would be fairly easy to unit test.

Reply:
I originally did exactly as you said (actually VS did it for me, to be exact) and faced multiple build errors, so I gave up changing the include structure of the files. I will try again.

Reply:
Actually, you are correct. I don't need to convert volumes, as the final result is so far removed from simple mathematical calculation it is essentially dimensionless anyway.

Reply:
Issue is solved now, but I can once again confirm that #include in consumption.cpp gives 3 linking errors during build.